apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/journalpost/EndreBruker.tsx

```diff
@@ -110,6 +110,11 @@ export const EndreBruker = ({
                     )}
                   </>
                 ),
+                error: (error) => (
+                  <Alert variant="error" size="small">
+                    {error.detail}
+                  </Alert>
+                ),
```

Comment:
Bruk heller `<ApiErrorAlert>{error.detail || 'Kunne ikke flytte journalpost til en annen bruker'}</ApiErrorAlert>`

Reply:
Vi garanterer at man får error fra våre routes, så `(error.detail || 'Hva som helst') === error.detail` (for våre egne endepunkt i kotlin-apper altså)